code/provision/playbooks/kubernetes-cluster.yml

```diff
@@ -32,6 +32,10 @@
     - role: kubernetes-namespace
       namespace: "{{ prod_namespace }}"
     - role: kubernetes-secrets
+    - role: vault-secrets
```

Comment:
These roles can't run until after Vault has been deployed so should be after the `datalabs-base` role in the `kubernetes-tools.yml` playbook.